packages/data-service/src/data-service.ts

```diff
@@ -370,7 +370,9 @@ class DataService extends EventEmitter {
 
     const listCollections = async () => {
       try {
-        return await db.listCollections(filter, { nameOnly }).toArray();
+        const collections = await db.listCollections(filter, { nameOnly }).toArray();
+        console.log('all collecitons', collections);
+        return collections;
```

Comment:
Debugging leftover? :)

Reply:
Just a draft that I pushed to be able to point at stuff while I'm working on it ;)